redisson/src/main/java/org/redisson/client/handler/CommandPubSubDecoder.java

```diff
@@ -222,24 +223,19 @@ protected MultiDecoder<Object> messageDecoder(CommandData<Object, Object> data,
                 return null;
             }
             return commandData.getCommand().getReplayMultiDecoder();
-        } else if ("message".equals(command)) {
+        } else if (TYPE_MESSAGES.contains(command)) {
             byte[] channelName = (byte[]) parts.get(1);
             PubSubEntry entry = entries.get(new ChannelName(channelName));
             if (entry == null) {
                 return null;
             }
             return entry.getDecoder();
-        } else if ("pmessage".equals(command)) {
-            byte[] patternName = (byte[]) parts.get(1);
-            PubSubEntry entry = entries.get(new ChannelName(patternName));
-            if (entry == null) {
-                return null;
-            }
-            return entry.getDecoder();
         } else if ("pong".equals(command)) {
             return new ListObjectDecoder<>(0);
         }
-
+        if (data == null) {
```

Comment:
what is the reason to add it?

Reply:
`data` can be null and `return data.getCommand().getReplayMultiDecoder();` may produce NullPointerException. Of course, it will handle but I think making exceptions is not the correct way.
Maybe I made a mistake, let me know.

Reply:
Exception in that case means that there is an issue command handling which should be evaluated and fixed. Using this code you suggest we won't see that.

Reply:
yes, you're right, thank you